marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -278,6 +282,28 @@ handleActionWithoutNavigationGuard =
         ( handleAction
         )
 
+carryMetadataAction ::
+  forall m.
+  MonadAff m =>
+  MonadAsk Env m =>
+  ME.MetadataAction ->
+  HalogenM State Action ChildSlots Void m Unit
+carryMetadataAction action = do
+  modifying (_contractMetadata) case action of
+    SetContractName name -> set _contractName name
+    SetContractType typeName -> set _contractType typeName
+    SetContractDescription description -> set _contractDescription description
+    SetRoleDescription tokenName description -> over _roleDescriptions $ Map.insert tokenName description
+    DeleteRoleDescription tokenName -> over _roleDescriptions $ Map.delete tokenName
+    SetSlotParameterDescription slotParam description -> over _slotParameterDescriptions $ Map.insert slotParam description
+    DeleteSlotParameterDescription slotParam -> over _slotParameterDescriptions $ Map.delete slotParam
+    SetValueParameterDescription valueParam description -> over _valueParameterDescriptions $ Map.insert valueParam description
+    DeleteValueParameterDescription valueParam -> over _valueParameterDescriptions $ Map.delete valueParam
+    SetChoiceDescription choiceName description -> over _choiceDescriptions $ Map.insert choiceName description
+    DeleteChoiceDescription choiceName -> over _choiceDescriptions $ Map.delete choiceName
+  metadata <- use _contractMetadata
```

Comment:
This is no longer needed, right?

Reply:
Oh, yes, thanks, unused variable, I am surprised it is not a warning